modules/compliance-timeout.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-timeout_{context}"]
+= Configuring ScanSetting timeout
+
+The `ScanSetting` object has a timeout option that can be specified in the `ComplianceScanSetting` object as a duration string, such as 1h30m. If the scan does not finish within the specified timeout, the scan will be reattempted until the `maxRetryOnTimeout` limit is reached.
```

Comment:
I know some people are weary of using the word "will" (as it can represent future tense). In the event you're one of those people, you could reword it as "the scan automatically reattempts until. . .", or just "reattempts. . ." 

Up to you. I don't mind it as-is. 

Reference: https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense